src/index.js

```diff
@@ -503,7 +513,7 @@ function createConfig(options, entry, format, writeMeta) {
 								},
 								files: options.entries,
 							},
-							tsconfig: options.tsconfig,
+							tsconfig: tsconfigPath,
```

Comment:
```suggestion
							tsconfig: options.tsconfig,
```

This is the default behaviour with rpt2.